### ISSUE TITLE
ubuntu 23.10 OS throwing this error

### ISSUE THREAD

**Issue by ratantirkey** [ifl0w/RandomWallpaperGnome3#177]
The settings of extension randomwallpaper@iflow.space had an error:
```
SyntaxError: import declarations may only appear at top level of a module
```

Stack trace:
```
@file:///home/ratan/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/prefs.js:4:15
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
```

**Comment by Draky50110** [comment]
Ubuntu 23.10 comes with Gnome 45 which breaks extensions.

See #172

**Comment by ifl0w** [comment]
It should be solved with v3.0.0.
